lib/Common/Core/ConfigParser.cpp

```diff
@@ -347,9 +347,63 @@ void ConfigParser::ParseRegistryKey(HKEY hk, CmdLineArgsParser &parser)
         }
     }
 
+    SetConfigStringFromRegistry(hk, _u("TelemetryDiscriminator1"), Js::Configuration::Global.flags.TelemetryDiscriminator1);
```

Comment:
We read from registry here so we have these values in retail builds.  Crawler will set these as part of the Crawler configuration.